addons/project_todo/views/project_task_views.xml

```diff
@@ -200,7 +199,7 @@
                                 <field name="name" display="full" class="w-100 text-truncate align-middle"/>
                             </span>
                         </div>
-                        <field name="user_ids" widget="many2many_avatar_user"/>
+                        <field name="user_ids" widget="many2many_avatar_user" domain="[]"/>
```

Comment:
I am not sure to understand why we add a domain attribute here.

Can you explain?

Thanks in advance! 🙂

Reply:
Unlike the other user_ids field, there we actually want to be able to add external/inactive user to the task activity. So we need to specify an empty domain so the default domain doesn't apply

Reply:
Sorry but I don't see why we want that behavior on that view. 😕

Reply:
Yep probably. 🙂 

thanks for the changes. 🙂